### PR TITLE
demo(sidenav): add accessibility demo pages

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

**Comment by mmalerba** [comment]
fixes #6501

File: src/demo-app/a11y/sidenav/dual-sidenav-a11y.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<mat-toolbar color="primary" role="header" aria-label="Dual sidenav app">
+  <button mat-icon-button aria-label="Navigation menu" (click)="startSnav.toggle()">
+    <mat-icon aria-hidden="true">menu</mat-icon>
+  </button>
+  <h1 class="a11y-demo-sidenav-app-name">Dual Sidenav App</h1>
+  <span class="a11y-demo-sidenav-spacer" aria-hidden="true"></span>
+  <button mat-icon-button aria-label="Playlist menu" (click)="endSnav.toggle()">
```
Comment (tinayuangao) [inline src/demo-app/a11y/sidenav/dual-sidenav-a11y.html]: Is there a way to focus on the playlist items when toggle sidenav? The focus stays in the toggle icon button, and user need to go through the left sidenav to reach the second sidenav.

**Comment by mmalerba** [comment]
done

File: src/demo-app/a11y/sidenav/mobile-sidenav-a11y.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  moduleId: module.id,
+  selector: 'mobile-sidenav-a11y',
+  templateUrl: 'mobile-sidenav-a11y.html',
+  styleUrls: ['shared.css', 'mobile-sidenav-a11y.css'],
+  host: {'class': 'a11y-demo-sidenav-app'},
+  encapsulation: ViewEncapsulation.None,
+  preserveWhitespaces: false,
+})
+export class SidenavMobileAccessibilityDemo implements OnDestroy {
+  mobileQuery = matchMedia('(max-width: 600px)');
```
Comment (andrewseguin) [inline src/demo-app/a11y/sidenav/mobile-sidenav-a11y.ts]: Looks like this is breaking internal build, where is this imported?

**Pull request merged**